build.psm1

```diff
@@ -466,18 +477,19 @@ Fix steps:
         Push-Location $Options.Top
 
         if ($Options.Runtime -notlike 'fxdependent*') {
-            if ($Options.Runtime -like 'win-arm*') {
-                $Arguments += "/property:SDKToUse=Microsoft.NET.Sdk"
-            } else {
-                $Arguments += "/property:SDKToUse=Microsoft.NET.Sdk.WindowsDesktop"
+            $sdkToUse = 'Microsoft.NET.Sdk'
+            if ($Options.Runtime -like 'win7-*' -and !$ForMinimalSize) {
```

Comment:
Maybe add a comment here to state that WPF / Winforms assemblies are included when `Microsoft.NET.Sdk.WindowsDesktop` is used.